dotnet/src/Autogen.Ollama/Agent/OllamaAgent.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// OllamaAgent.cs
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoGen.Core;
+
+namespace Autogen.Ollama;
+
+public class OllamaAgent : IStreamingAgent
```

Comment:
Is the `OllamaAgent` essentially be
`OllamaClientAgent` with `OllamaMessageConnector`? If so we can just remove this class?

Reply:
yes of course